video/out/gpu/context.c

```diff
@@ -182,6 +186,11 @@ struct ra_ctx *ra_ctx_create(struct vo *vo, const char *context_type,
         };
 
         MP_VERBOSE(ctx, "Initializing GPU context '%s'\n", ctx->fns->name);
+        if ((strcmp(ctx->fns->name, "wayland") == 0 || strcmp(ctx->fns->name, "waylandvk") == 0) && gnome) {
```

Comment:
`strstr("wayland")` perhaps? Also, this now physically prevents wayland being used on gnome at all, rather than just skipping it during probing. Need to  additionally condition on probing. if the context is forced, we still want to be able to use it.

Reply:
Oh crap, I thought that line was only executed during probing. My bad.